pages/portfolio.html

```diff
@@ -1,12 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles/portfolio.css">
 </head>
+
 <body>
-  
+    <header>
+
+        <h1>Ashley Benson</h1>
+        <nav>
+            <ul>
+                <li><a href="http://127.0.0.1:5500/pages/index.html">Home</a></li>
+                <li><a href="http://127.0.0.1:5500/pages/portfolio.html">Portfolio</a></li>
+                <li><a href="http://127.0.0.1:5500/pages/about.html">About</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <h2>Viewing Party</h2>
+            <p>Created a platform that friends can use to keep track of and exchange movies to watch. <a href="https://github.com/bensonaa1988/viewing-party.git">Find the GitHub repo here</a>.</p>
+            <ul class="tags">
+                <li>Python</li>
+                <li>TDD</li>
+
+            </ul>
+        </section>
+        <section>
+            <h2>Hello Books</h2>
+            <p>My first API! Created a database to store book information. <a href="https://github.com/bensonaa1988/hello-books-api.git">Find the GitHub repo here</a>.</p>
```

Comment:
With the lack of `text-decoration` on the anchor, it's hard to tell that there are links here. Users expect links to have underlines, and we shouldn't subvert this expectation without very good reason.